libraries/networking/src/EntityScriptClient.cpp

```diff
@@ -15,9 +16,12 @@ GetScriptStatusRequest::~GetScriptStatusRequest() {
 
 void GetScriptStatusRequest::start() {
     auto client = DependencyManager::get<EntityScriptClient>();
-    qDebug() << "Sending script status request";
-    client->getEntityServerScriptStatus(_entityID, [this](bool responseReceived) {
-        qDebug() << "Received script status request";
+    client->getEntityServerScriptStatus(_entityID, [this](bool responseReceived, bool isRunning, EntityScriptStatus status, QString errorInfo) {
+        this->responseReceived = responseReceived;
+        this->isRunning = isRunning;
+        this->status = status;
+        this->errorInfo = errorInfo;
```

Comment:
I hate `this->`. There's only a single reason to ever use it and this is not it =p